pkg/trace/traceutil/normalize.go

```diff
@@ -113,11 +115,21 @@ func fallbackService(lang string) string {
 
 const maxTagLength = 200
 
-// NormalizeTag applies some normalization to ensure the tags match the backend requirements.
+// NormalizeTag applies some normalization to ensure the full tag_key:tag_value string matches the backend requirements.
 func NormalizeTag(v string) string {
+	return normalize(v, true)
+}
+
+// NormalizeTagValue applies some normalization to ensure the tag value matches the backend requirements.
+// It should be used for cases where we have just the tag_value as the input (instead of tag_key:tag_value).
+func NormalizeTagValue(v string) string {
+	return normalize(v, false)
```

Comment:
The requirement to not start with a digit is on the tag key: https://docs.datadoghq.com/getting_started/tagging/#define-tags. I read fast but you normalize service name with this method, which disallows starting with a digit. Can you help clarify as maybe Im misunderstanding the goal of the change?